vips/image.go

```diff
@@ -1235,6 +1235,16 @@ func (r *ImageRef) FindTrim(threshold float64, backgroundColor *Color) (int, int
 	return vipsFindTrim(r.image, threshold, backgroundColor)
 }
 
+// GetPoint reads a single pixel on an image.
+// The pixel values are returned in a slice of length n.
+func (r *ImageRef) GetPoint(x int, y int) ([]float64, error) {
```

Comment:
why `GetPoint` vs `GetValue` or `GetColor`? I guess it mimicks vips... Just curious if you thought of this

Reply:
Maybe GetColor is more of RGB and may not necessary cover alpha channel? GetValue sounds too ambiguous?
Since this is a libvips binding, I guess it would be better name it as close as possible?
Seems others are naming it the same way too https://libvips.github.io/pyvips/vimage.html#pyvips.Image.getpoint

Reply:
Maybe GetColor is more of RGB and may not necessary cover alpha channel? GetValue sounds too ambiguous?
Since this is a libvips binding, I guess it would be better name it as close as possible?
Seems others are naming it the same way too https://libvips.github.io/pyvips/vimage.html#pyvips.Image.getpoint